### DISCUSSION TITLE
SFTP with key not keeping settings

### DISCUSSION THREAD

**Discussion by geejaytee** [iterate-ch/cyberduck#15394] [Q&A] [answered]
I have a server that requires an SSH key, which I use successfully for logging in via SSH on the command line. However, I have set the server login using the key and the password as a bookmark, but Cyberduck doesn't log in on the first attempt - I have to manually set the correct key and password after the first log in fails. Why is this?

(Mac OS X 14.1.1, Cyberduck - Version 8.7.1 (40770))

**Comment by tatze96** [comment]
Can you please [enable debug logging](https://docs.cyberduck.io/cyberduck/support/#debug-logging-level), re-create the connection issue, and send us the log file afterward? If you don't like to see sensitive data to be shared publicly, please send the log file with the ticket reference GH-15394 to support@cyberduck.io, otherwise attach it here.

**Sidenote**: If you want to share the log file here, make sure to attach it through the GitHub web interface, as it wouldn't be visible to us otherwise.

**Reply by iKlsR** [reply]
Just saw that this discussion was linked, I believe it's the same issue here https://github.com/iterate-ch/cyberduck/issues/15156, I have a clean log attached there starting from app launch to the steps outlined.

**Reply by AliveDevil** [reply, marked as answer]
Reproduced, and notified user in internal issue tracker.

Amended original ticket by @iKlsR in #15156.
